e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -742,7 +742,7 @@ if ($InstallPrivatePreviewResources)
     }
     catch
     {
-        Write-Host "`nError while linking DPS $dpsName to Trust Bundle $dpsTrustBundleId."
+        Write-Host -ForegroundColor Red "`nError while linking DPS $dpsName to Trust Bundle $dpsTrustBundleId."
         Write-Error -Message $_.ErrorDetails.Message
```

Comment:
The script is set to fail if the DPS-CA/Trust Bundle linking fails. The error message is highlighted in red and one possible troubleshooting step is highlighted above in yellow.